types/fabric/fabric-impl.d.ts

```diff
@@ -41,12 +41,18 @@ export function loadSVGFromString(
 /**
  * Takes url corresponding to an SVG document, and parses it into a set of fabric objects.
  * Note that SVG is fetched via XMLHttpRequest, so it needs to conform to SOP (Same Origin Policy)
- * @param [reviver] Method for further parsing of SVG elements, called after each fabric object created.
+ * @param {String} url URL to get a SVG from
+ * @param {Function} callback Callback is invoked when svg has been loaded
+ * @param {Function} [reviver] Method for further parsing of SVG elements, called after each fabric object created.
+ * @param {Object} [options] options for crossOrigin
+ * @param {String} [options.crossOrigin] crossOrigin settings
+ *
  */
 export function loadSVGFromURL(
     url: string,
     callback: (results: Object[], options: any) => void,
     reviver?: Function,
+    options?: Object | string
```

Comment:
This just cannot be `string`, used as when passed down:
```js
 options.crossOrigin = parsingOptions && parsingOptions.crossOrigin;
```
http://fabricjs.com/docs/fabric.js.html#line4015
Most probably something like (the interface or type would be better instead of inline btw):

```suggestion
    options?: { crossOrigin?: string; }
```

Reply:
Oops I got a miss understanding 
below two comment in Document
http://fabricjs.com/docs/fabric.js.html#line4395
http://fabricjs.com/docs/fabric.js.html#line4396